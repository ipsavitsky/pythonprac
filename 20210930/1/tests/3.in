635464356,123,4,12,34,1,4,321,12