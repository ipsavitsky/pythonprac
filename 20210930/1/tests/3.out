[1, 4, 4, 123, 635464356, 321, 12, 12, 34]
